experiments/axon_geometry/notebooks/examine_neurons.ipynb

```diff
@@ -0,0 +1,392 @@
+    "        if os.path.exists(trace_data_path) is True:\n",
+    "            trace_data = np.load(trace_data_path, allow_pickle=True)\n",
+    "            print(\"Loaded segment {}\".format(i))\n",
+    "\n",
+    "            for node in trace_data:\n",
+    "                seg_length = node[\"seg_length\"]\n",
+    "                height = node[\"height\"]\n",
+    "                _class = node[\"class\"]\n",
+    "                mean_curvature = node[\"mean_curvature\"]\n",
+    "                mean_torsion = node[\"mean_torsion\"]\n",
+    "                \n",
+    "                log_seg_length = np.log10(seg_length)\n",
+    "\n",
+    "                log_mean_curvature = np.log10(mean_curvature)\n",
+    "                df = df.append({\"seg_id\": i, \"height\": height, \"class\": _class, \"log_seg_length\": log_seg_length, \"measure\": \"curvature\", \"value\": mean_curvature, \"log_value\": log_mean_curvature}, ignore_index=True)\n",
+    "\n",
+    "                log_mean_torsion = np.log10(mean_torsion)\n",
+    "                df = df.append({\"seg_id\": i, \"height\": height, \"class\": _class, \"log_seg_length\": log_seg_length, \"measure\": \"torsion\", \"value\": mean_torsion, \"log_value\": log_mean_torsion}, ignore_index=True)\n",
+    "    df.to_csv(df_path)\n",
+    "df[\"class\"] = df.apply(numerical_class, axis=1)\n",
+    "df[\"height_class\"] = df.apply(classify_height, axis=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Count number of neurons that fall into different orderings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stderr",
+     "text": [
+      "100%|██████████| 300/300 [00:01<00:00, 191.72it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "measures = [\"curvature\", \"torsion\"]\n",
+    "\n",
+    "classes = {0: \"primary\", 1: \"collateral\", 2: \"terminal\"}\n",
+    "n = len(classes)\n",
+    "data = np.zeros((2,3))\n",
+    "results = np.zeros((6,6))\n",
+    "\n",
+    "matrix_pairs = np.triu(np.ones((n, n)), k=1)\n",
+    "(coord_pairs_x, coord_pairs_y) = np.where(matrix_pairs == 1)\n",
+    "\n",
+    "example1_idxs = []\n",
+    "example2_idxs = []\n",
+    "\n",
+    "for i in tqdm(np.arange(0, max_id)):\n",
+    "    sample_query = df.loc[(df['seg_id'] == i)]\n",
+    "    num_segments = len(sample_query.index)\n",
+    "    if num_segments == 0:\n",
+    "        continue\n",
+    "\n",
+    "    for row, measure in enumerate(measures):\n",
+    "        for col, clas in enumerate(classes.keys()):\n",
+    "            sample_query = df.loc[(df['seg_id'] == i) & (df['class'] == clas) & (df['measure'] == measure)]\n",
+    "            value = np.mean(sample_query[\"value\"].to_numpy())\n",
+    "            data[row, col] = value\n",
```

Comment:
```suggestion
    "        for col, class in enumerate(classes.keys()):\n",
    "            sample_query = df.loc[(df['seg_id'] == i) & (df['class'] == class) & (df['measure'] == measure)]\n",
    "            value = np.mean(sample_query[\"value\"].to_numpy())\n",
    "            data[row, col] = value\n",
```

Reply:
class is a keyword, so let's stick with clas?